packages/core/test/integration/model/findAll.test.js

```diff
@@ -1377,7 +1377,7 @@ The following associations are defined on "Worker": "ToDos"`);
 
         const users = await this.User.findAll({
           attributes: [[
-            literal('(SELECT 7)'),
+            literal('(SELECT 7 AS X)'), // AS X for db2 compat
```

Comment:
@WikiRik do you know any expression I can put to make it compatible with db2?
Never used that db, asked chat gpt and told me to put that but it lied to me haha

Reply:
You need to specify a table (`FROM SYSIBM.SYSDUMMY1`)